postmortem-template.md

```diff
@@ -0,0 +1,51 @@
+# YYYY-MM-DD Postmortem title
+Date:
+Authors:
+Status:
+---
+
+## Summary
+*example:*
+```
+- Product - Caseflow, all products
```

Comment:
I think we can leave "Caseflow" out, this could be "all products" and lower down it could just say "users"

Reply:
oops, updated!